include/grpc/grpc_security_constants.h

```diff
@@ -29,6 +29,15 @@ extern "C" {
 #define GRPC_X509_CN_PROPERTY_NAME "x509_common_name"
 #define GRPC_X509_SAN_PROPERTY_NAME "x509_subject_alternative_name"
 #define GRPC_X509_PEM_CERT_PROPERTY_NAME "x509_pem_cert"
+// Please note that internally, we just faithfully pass whatever value we got by
+// calling SSL_get_peer_certificate() in OpenSSL/BoringSSL. This will mean in
```

Comment:
Actually, the code calls `SSL_get_peer_cert_chain()`.

Reply:
Sorry, I was referring to the wrong link. Thanks for pointing it out!